openverse_catalog/dags/providers/provider_api_scripts/museum_victoria.py

```diff
@@ -137,23 +137,26 @@ def _get_image_data(
 
     @staticmethod
     def _get_creator(media) -> str | None:
-        creators = media.get("creators", None)
+        creators = media.get("creators")
         if isinstance(creators, list):
             creators = ",".join(media.get("creators"))
         return creators
 
     @staticmethod
-    def _get_metadata(obj):
-        def join_string_list(object_key, obj):
-            data = obj.get(object_key)
-            return ",".join(data) if isinstance(data, list) else None
+    def join_string_list(object_key, obj):
+        data = obj.get(object_key)
+        return ",".join(data) if isinstance(data, list) else None
 
+    @staticmethod
+    def _get_metadata(obj):
         meta_data = {
             "datemodified": obj.get("dateModified"),
             "category": obj.get("category"),
             "description": obj.get("physicalDescription"),
-            "keywords": join_string_list("keywords", obj),
-            "classifications": join_string_list("classifications", obj),
+            "keywords": VictoriaDataIngester.join_string_list("keywords", obj),
+            "classifications": VictoriaDataIngester.join_string_list(
```

Comment:
Note for `self` here as well 😄

Reply:
This is a `static` method that doesn't have a `self` 😢

Reply:
D'oh! Sorry about that 😭